EssentialApp/EssentialApp/SceneDelegate.swift

```diff
@@ -39,6 +39,10 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 		LocalFeedImageDataLoader(store: store)
 	}()
 	
+	private lazy var navigationController: UINavigationController? = {
+		window?.rootViewController as? UINavigationController
+	}()
```

Comment:
Create the navigation controller as a lazy var so you don't need to unwrap it from the window hierarchy and it won't need to be optional.

```suggestion
	private lazy var navigationController = UINavigationController(
			rootViewController: FeedUIComposer.feedComposedWith(
				feedLoader: makeRemoteFeedLoaderWithLocalFallback,
				imageLoader: makeLocalImageLoaderWithRemoteFallback,
				onSelect: onFeedImageSelect))
```